packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_action_menu.test.tsx

```diff
@@ -53,26 +53,35 @@ describe('HeaderActionMenu', () => {
       return unmount;
     };
 
-  it('mounts the current value of the provided observable', async () => {
-    component = mount(<HeaderActionMenu actionMenu$={menuMount$} />);
+  it('mounts the current value of the provided observable', () => {
+    const TestComponent = () => {
+      const mounter = useHeaderActionMenuMounter(menuMount$);
+      return <HeaderActionMenu mounter={mounter} />;
+    };
+    component = mount(<TestComponent />);
 
     act(() => {
       menuMount$.next(createMountPoint('FOO'));
     });
-    await refresh();
```

Comment:
Minor cleanup: I've removed the unnecessary `await` statements, as they create warnings in my editor.